packages/backend/src/models/CatalogModel/CatalogModel.ts

```diff
@@ -89,14 +165,12 @@ export class CatalogModel {
                 `${CatalogTableName}.name`,
                 'description',
                 'type',
-                {
-                    search_rank: searchRankRawSql ?? 0,
-                },
                 `${CachedExploreTableName}.explore`,
                 `required_attributes`,
                 `chart_usage`,
                 // Add tags as an aggregated JSON array
                 {
+                    search_rank: searchRankRawSql ?? 0,
```

Comment:
can you explain this change pls?

Reply:
It was just to pass it in the same object as catalog_tags, no real reason why making it smaller